core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -332,4 +332,17 @@ default boolean indexExists(String namespace, String table, String columnName)
   default boolean tableExists(String namespace, String table) throws ExecutionException {
     return getNamespaceTableNames(namespace).contains(table);
   }
+
+  /**
+   * Repair a table which may be in an unknown state
+   *
+   * @param namespace an existing namespace
+   * @param table an existing table
+   * @param metadata the metadata associated to the table to repair
+   * @param options options to repair
+   * @throws ExecutionException if the operation failed
+   */
+  void repairTable(
+      String namespace, String table, TableMetadata metadata, Map<String, String> options)
+      throws ExecutionException;
```

Comment:
The definition of the new `repairTable` method available on all Admin interfaces.

Reply:
So, it's repairing the metadata of tables, should it be `repairTableMetadata` or `repairMetadataTable`?
`repairTable` sounds like repairing a table itself like Cassandra's repair.